module/applications/components/item-list-controls.mjs

```diff
@@ -1,3 +1,16 @@
+/**
+ * Sort constants
+ */
+const sortIcons = {
+  a: "fa-arrow-down-a-z",
+  p: "fa-arrow-down-1-9",
+  m: "fa-arrow-down-short-wide"
+};
+const sortLocalisations = {
+  a: "Alpha",
+  p: "Priority",
+  m: "Manual"
+};
```

Comment:
We can place these inside the class, as static properties, to allow subclasses to override them.

```js
/**
 * @typedef {object} SortModeConfiguration5e
 * @property {string} icon
 * @property {string} label
 * @property {ItemListComparator5e} comparator
 */

/**
 * @callback ItemListComparator5e
 * @param {object} a
 * @param {object} b
 * @returns {number}
 */

/**
 * Sort mode configuration.
 * @type {Record<string, SortModeConfiguration5e>}
 */
static SORT_MODES = {
  a: {
    icon: "fa-arrow-down-a-z",
    label: "SIDEBAR.SortModeAlpha",
    comparator: (a, b) => a.name.localeCompare(b.name, game.i18n.lang)
  },
  p: {
    icon: "fa-arrow-down-1-9",
    label: "SIDEBAR.SortModePriority",
    comparator: (a, b) => (a.preparationMode - b.preparationMode)
      || (a.prepared - b.prepared)
      || a.name.localeCompare(b.name, game.i18n.lang)
  },
  m: {
    icon: "fa-arrow-down-short-wide",
    label: "SIDEBAR.SortModeManual",
    comparator: (a, b) => a.sort - b.sort
  }
};
```